subprojects/gradle/build-checks/src/main/kotlin/com/avito/android/build_checks/internal/unique_app_res/UniqueAppResourcesTaskCreator.kt

```diff
@@ -1,3 +1,6 @@
+// todo use new agp api
+@file:Suppress("DEPRECATION")
```

Comment:
Could you please mark it in more narrow scope instead of the whole file?

Reply:
done